lib/console.js

```diff
@@ -162,6 +162,25 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
+Console.prototype.dirxml = function dirxml(...data) {
+  const optionProps = ['showHidden', 'depth', 'colors'],
+    maybeOptions = Object.getOwnPropertyNames(data.slice(-1)),
```

Comment:
These should be separate `const` statements.

Reply:
Noted!